src/main/scala/com/softwaremill/mqperf/mq/RedisStreamsMq.scala

```diff
@@ -0,0 +1,54 @@
+package com.softwaremill.mqperf.mq
+
+import com.softwaremill.mqperf.config.TestConfig
+import com.typesafe.scalalogging.StrictLogging
+import redis.clients.jedis.StreamEntryID.UNRECEIVED_ENTRY
+import redis.clients.jedis.params.XReadGroupParams
+import redis.clients.jedis.{HostAndPort, JedisPool, StreamEntryID}
+
+import java.util.UUID.randomUUID
+import scala.collection.JavaConverters._
+
+class RedisStreamsMq(testConfig: TestConfig) extends Mq with StrictLogging{
+  override type MsgId = StreamEntryID
+
+  final val StreamId = "mystream"
```

Comment:
[minor] you usually don't need `final` here, unless you want to explicitly prohibit overriding. It's an immutable value either way